### PR TITLE
Add conclusion scene

### LINKED ISSUE #47: Add gdscript file with the dialogic code for the Conclusion Phase
Add a gdscript which implements the section of the story here: https://docs.google.com/document/d/1CX4Qqq5iBo791zUdzjrONPgscOvLL3iLSmWALtFGDCY/edit#heading=h.k50j9zpbsrsq

Have the player pick a suspect for the murder.

### PR DESCRIPTION
**Pull request by brydenli**
Resolves #47

### CODE REVIEW

File: src/scenes/conclusion/conclusion.gd [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+func _ready() -> void:
+    var dialog = Dialogic.start("conclusion0");
+    add_child(dialog);
+    dialog.connect("timeline_end", self, "end_conclusion");
+
+func end_conclusion(_arg) -> void:
+    var culprit_choice = Dialogic.get_variable("CulpritChoice");
+    if (culprit_choice == "Bunny"):
+        WinLossFlag.set_flag(true);
+        print(WinLossFlag.get_flag());
```
Comment (ColinT) [inline src/scenes/conclusion/conclusion.gd]: Remove `print()`

File: src/autoloads/win-loss-flag.gd [hunk 2]
```diff
@@ -0,0 +1,18 @@
+extends Node
+
+var WIN = true;
+var LOSS = false;
```
Comment (ColinT) [inline src/autoloads/win-loss-flag.gd]: Either use an actual enum and pass enum values, or only use booleans.

**Review by ColinT — CHANGES_REQUESTED** [review]

File: src/scenes/conclusion/conclusion.gd (hunk 1 above)
Comment (brydenli) [reply src/scenes/conclusion/conclusion.gd]: Updated 👍

File: src/autoloads/win-loss-flag.gd (hunk 2 above)
Comment (brydenli) [reply src/autoloads/win-loss-flag.gd]: Updated 👍

Comment (brydenli) [reply src/autoloads/win-loss-flag.gd]: Updated to use an enum for the flag conditions

**Review by anson-li — APPROVED** [review]

**Pull request merged**